src/sql/platform/dialog/customDialogService.ts

```diff
@@ -8,14 +8,17 @@
 import * as sqlops from 'sqlops';
 import { OptionsDialog } from 'sql/base/browser/ui/modal/optionsDialog';
 import { DialogModal } from 'sql/platform/dialog/dialogModal';
-import { Dialog } from 'sql/platform/dialog/dialogTypes';
+import { WizardModal } from 'sql/platform/dialog/wizardModal';
+import { Dialog, Wizard, DialogTab } from 'sql/platform/dialog/dialogTypes';
 import { IModalOptions } from 'sql/base/browser/ui/modal/modal';
 import { IInstantiationService } from 'vs/platform/instantiation/common/instantiation';
 
-const defaultOptions: IModalOptions = { hasBackButton: true, isWide: false };
+const defaultOptions: IModalOptions = { hasBackButton: false, isWide: false };
+const defaultWizardOptions: IModalOptions = { hasBackButton: false, isWide: true };
```

Comment:
I see there is also a `previous` button which appears to do what I thought `back` would do.  It may be worth thinking up a different name for `back`.  I assume it's the exit dialog button at the top of dialog?  Maybe `closeButton` is less ambiguous?

Reply:
Yes, this is the exit arrow at the top of the dialog. I've added a note in the ui extensibility task #559 to rename it